<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLedText</class>
 <widget class="QWidget" name="frmLedText">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="LedText" name="ledText1"/>
   </item>
   <item>
    <widget class="LedText" name="ledText2"/>
   </item>
   <item>
    <widget class="LedText" name="ledText3"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LedText</class>
   <extends>QWidget</extends>
   <header>ledtext.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
